XRTK-Core/Packages/com.xrtk.core/Utilities/ValidateConfiguration.cs

```diff
@@ -158,5 +179,73 @@ public static bool ValidateControllerProfiles(Type[] mappingTypesToValidate)
 
             return false;
         }
+
+        /// <summary>
+        /// Controller Mapping function to test for a controller mappings for a specific hand
+        /// </summary>
+        /// <param name="mappingTypesToValidate">Array of controller mappings to validate</param>
+        /// <param name="controllerHandedness">The <see cref="Handedness"/> of the controller to validate</param>
+        /// <returns></returns>
+        public static bool ValidateControllerMappings(Type[] mappingTypesToValidate, Handedness controllerHandedness)
```

Comment:
Couldn't this just be an override?

It seems a lot of this is duplicated from above.

Reply:
No, because one looks at data providers, one looks at mapping profiles and the other looks at mapping interactions.